adam/visualization/panda3d_interface.py

```diff
@@ -60,7 +59,7 @@ def __init__(self) -> None:
 
         self.ground_plane = self._load_model("ground.egg")
         self.ground_plane.reparentTo(self.render)
-        self.ground_plane.setPos(0, 0, -1)
+        self.ground_plane.setPos(0, 0, -2)
```

Comment:
<img width="802" alt="Screen Shot 2019-12-11 at 15 51 56" src="https://user-images.githubusercontent.com/55997224/70659498-3581e500-1c2e-11ea-981c-c322541b11a2.png">It should be -1 not -2, but it is offset like that so that a 2x2x2 cube positioned with its center point at (0,0,0) rests on the ground. Maybe that isn't the best assumption to make, but that's the idea.

Reply:
Can you alter it to have the ground plane at `z=0`? It is much less confusing that way, and also fits the "gravity" in our positioning model.

Reply:
Yes. I might have moved it there however long ago not understanding that the plane was changing color based on whether it was above a point light that was (until the upcoming changes) sitting at 0 in the Z dimension. In the screenshot above, the plane is unlit because it is blocking the light source!